apps/web/src/config/constants/endpoints.ts

```diff
@@ -10,7 +10,8 @@ export const SNAPSHOT_API = `${SNAPSHOT_BASE_URL}/graphql`
 export const SNAPSHOT_HUB_API = `${SNAPSHOT_BASE_URL}/api/message`
 export const GRAPH_API_POTTERY = 'https://api.thegraph.com/subgraphs/name/pancakeswap/pottery'
 // export const ONRAMP_API_BASE_URL = 'https://pcs-onramp-api.com'
-export const ONRAMP_API_BASE_URL = 'https://onramp2-api.pancakeswap.com'
+export const ONRAMP_API_BASE_URL = 'https://monkfish-app-s4mda.ondigitalocean.app'
```

Comment:
why are we using a dev endpoint here?

Reply:
because the develop branch hasnt been merged to master. thats what this PR is mainly for. we use devlop api on mobile. this pr just syncs the two.the api will be merged first will swap out then

Reply:
Let's convert to draft until the prod api is ready